server/lib/activity-event.js

```diff
@@ -0,0 +1,35 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var _ = require('lodash');
+
+var ALLOWED_QUERY_PARAMS = [
+  'context',
+  'entrypoint',
+  'migration',
+  'service',
+  'utm_campaign',
+  'utm_content',
+  'utm_medium',
+  'utm_source',
+  'utm_term'
+];
+
+module.exports = function (event, data, request) {
+  var eventData = _.assign({
```

Comment:
Oh, that's an excellent point and I think @vladikoff raised it in the previous PR too. I need to fix it, sorry.